ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1763778&tab=buildLog) on [master@6d541881b9fc71c36175814fb206487d46b87f1a](https://github.com/cockroachdb/cockroach/commits/6d541881b9fc71c36175814fb206487d46b87f1a):

```
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/opt/optbuilder/select.go:822 +0x853
github.com/cockroachdb/cockroach/pkg/sql/opt/optbuilder.(*Builder).buildStmt(0xc00119e480, 0x71cd1e0, 0xc001327090, 0x0, 0x0, 0x0, 0xc001799400, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/opt/optbuilder/builder.go:253 +0x2c0
github.com/cockroachdb/cockroach/pkg/sql/opt/optbuilder.(*Builder).buildStmtAtRoot(0xc00119e480, 0x71cd1e0, 0xc001327090, 0x0, 0x0, 0x0, 0xc001799400, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/opt/optbuilder/builder.go:217 +0x127
github.com/cockroachdb/cockroach/pkg/sql/opt/optbuilder.(*Builder).Build(0xc00119e480, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/opt/optbuilder/builder.go:188 +0x43f
github.com/cockroachdb/cockroach/pkg/sql.(*optPlanningCtx).buildExecMemo(0xc001960ab0, 0x71b3d60, 0xc001290c60, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/plan_opt.go:417 +0x676
github.com/cockroachdb/cockroach/pkg/sql.(*planner).makeOptimizerPlan(0xc0019603c8, 0x71b3d60, 0xc001290c60, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/plan_opt.go:159 +0xae
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).makeExecPlan(0xc001960000, 0x71b3d60, 0xc001290c60, 0xc0019603c8, 0x6006eb0, 0xc001914048)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:731 +0x190
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).dispatchToExecutionEngine(0xc001960000, 0x71b3d60, 0xc001290c60, 0xc0019603c8, 0x720c6a0, 0xc000d31b60, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:624 +0x1dc
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmtInOpenState(0xc001960000, 0x71b3d60, 0xc001290c60, 0x71cd1e0, 0xc001327090, 0x643ac69, 0x140, 0x0, 0x2, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:412 +0xdbb
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt(0xc001960000, 0x71b3d60, 0xc001290c60, 0x71cd1e0, 0xc001327090, 0x643ac69, 0x140, 0x0, 0x2, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:93 +0x6e1
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd(0xc001960000, 0x71b3ca0, 0xc0012c8600, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1315 +0x2357
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run(0xc001960000, 0x71b3d60, 0xc001290810, 0xc000925e00, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1244 +0x484
github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).initConnEx.func1(0xc001960000, 0x71b3d60, 0xc001290810, 0xc000974480, 0xc00024fbc0, 0xc000a42990, 0xc0013e06e4)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:158 +0x8f
created by github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).initConnEx
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:157 +0x5be

rax    0x0
rbx    0x15450d57f000
rcx    0x15450ba6f428
rdx    0x6
rdi    0x6bf4
rsi    0x6c08
rbp    0x73b4033
rsp    0x154505ffce98
r8     0xfffffffffffffd40
r9     0xfefefefefefeff09
r10    0x8
r11    0x202
r12    0x175
r13    0x73b3f7e
r14    0xc0001ee320
r15    0x10
rip    0x15450ba6f428
rflags 0x202
cs     0x33
fs     0x0
gs     0x0
FAIL	github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl	0.835s

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1813334&tab=buildLog) on [master@b6d26e2280d48d9f8bfb8a37965f00f0415f47c0](https://github.com/cockroachdb/cockroach/commits/b6d26e2280d48d9f8bfb8a37965f00f0415f47c0):

```
I200317 13:26:18.626772 20781 kv/kvserver/replica_command.go:397  [n1,s1,r63/1:/Table/5{8-9/1}] initiating a split of this range at key /Table/58/1 [r66] (manual)
I200317 13:26:19.225701 20781 kv/kvserver/replica_command.go:397  [n1,s1,r63/1:/Table/5{8-9}] initiating a split of this range at key /Table/58/1 [r67] (manual)
I200317 13:26:19.231871 20781 kv/kvserver/replica_command.go:397  [n1,s1,r67/1:/Table/5{8/1-9}] initiating a split of this range at key /Table/58/2 [r68] (manual)
I200317 13:26:19.370321 19402 gossip/gossip.go:1527  [n1] node has connected to cluster via gossip
I200317 13:26:19.372194 19402 kv/kvserver/stores.go:266  [n1] wrote 0 node addresses to persistent storage
I200317 13:26:20.281523 20778 jobs/registry.go:764  [n1] job 538644242736414721: stepping through state succeeded with error <nil>
I200317 13:26:20.314110 20778 jobs/registry.go:916  [n1] job 538644242736414721: status succeeded after adoption finished
I200317 13:26:20.330591 21898 kv/kvserver/replica_command.go:397  [n1,split,s1,r45/1:/Table/5{5/2-6/1}] initiating a split of this range at key /Table/56 [r69] (zone config)
I200317 13:26:20.465981 21470 kv/kvserver/replica_command.go:397  [n1,s1,r40/1:/Table/55/{1-2}] initiating a split of this range at key /Table/55/1/0/1/1 [r70] (manual)
I200317 13:26:20.528175 21461 kv/kvserver/replica_command.go:397  [n1,s1,r41/1:/Table/61/{1-2}] initiating a split of this range at key /Table/61/1/0/1 [r71] (manual)
I200317 13:26:20.707795 20756 jobs/registry.go:764  [n1] job 538644242729271297: stepping through state succeeded with error <nil>
I200317 13:26:20.790535 20781 jobs/registry.go:764  [n1] job 538644242744344577: stepping through state succeeded with error <nil>
I200317 13:26:20.839318 20781 jobs/registry.go:916  [n1] job 538644242744344577: status succeeded after adoption finished
I200317 13:26:20.840619 20722 jobs/registry.go:764  [n1] job 538644242718654465: stepping through state succeeded with error <nil>
I200317 13:26:20.850911 20584 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 3.280429886s, 0.46 MiB/s)
I200317 13:26:20.870190 20756 jobs/registry.go:916  [n1] job 538644242729271297: status succeeded after adoption finished
I200317 13:26:20.878171 20722 jobs/registry.go:916  [n1] job 538644242718654465: status succeeded after adoption finished
I200317 13:26:20.888484 20583 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 3.318523892s, 1.29 MiB/s)
I200317 13:26:20.893532 20582 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 3.323340369s, 5.28 MiB/s)
I200317 13:26:21.289215 20825 jobs/registry.go:764  [n1] job 538644242767708161: stepping through state succeeded with error <nil>
I200317 13:26:21.301543 20825 jobs/registry.go:916  [n1] job 538644242767708161: status succeeded after adoption finished
I200317 13:26:21.302002 20587 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 3.718985569s, 8.54 MiB/s)
I200317 13:26:21.732672 21588 kv/kvserver/replica_command.go:397  [n1,s1,r47/1:/Table/60/{1-2}] initiating a split of this range at key /Table/60/1/0/1/-3001/1 [r72] (manual)
I200317 13:26:22.396562 20822 jobs/registry.go:764  [n1] job 538644242795134977: stepping through state succeeded with error <nil>
I200317 13:26:22.407152 20822 jobs/registry.go:916  [n1] job 538644242795134977: status succeeded after adoption finished
I200317 13:26:22.407557 20588 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 4.823625958s, 4.50 MiB/s)
I200317 13:26:22.407787 19348 util/stop/stopper.go:539  quiescing
W200317 13:26:22.407922 22617 kv/kvserver/intentresolver/intent_resolver.go:732  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (5.15s)
        all_test.go:95: sql: no rows in result set
            importing table item
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1826291&tab=buildLog) on [master@6c3c5d4a9a0b837538a558fa22dcd4ff712fd8f5](https://github.com/cockroachdb/cockroach/commits/6c3c5d4a9a0b837538a558fa22dcd4ff712fd8f5):

```
I200325 05:20:30.612331 20654 kv/kvserver/replica_command.go:397  [n1,s1,r73/1:/Table/5{8/1-9}] initiating a split of this range at key /Table/58/2 [r75] (manual)
I200325 05:20:30.810925 20727 kv/kvserver/replica_command.go:397  [n1,s1,r69/1:/Table/{59-60}] initiating a split of this range at key /Table/59/1 [r76] (manual)
I200325 05:20:30.980993 20985 kv/kvserver/replica_command.go:397  [n1,s1,r37/1:/Table/54/{1-2}] initiating a split of this range at key /Table/54/1/0/1/1 [r77] (manual)
I200325 05:20:31.230075 20727 jobs/registry.go:764  [n1] job 540813651376701441: stepping through state succeeded with error <nil>
I200325 05:20:31.255569 19385 kv/kvserver/queue.go:578  [n1,s1] rate limited in MaybeAdd (split): throttled on async limiting semaphore
I200325 05:20:31.380844 20727 jobs/registry.go:916  [n1] job 540813651376701441: status succeeded after adoption finished
I200325 05:20:31.421267 19678 ccl/workloadccl/fixture.go:446  imported 114 KiB in new_order table (9000 rows, 0 index entries, took 2.909649402s, 0.04 MiB/s)
I200325 05:20:31.553787 20692 jobs/registry.go:764  [n1] job 540813651287015425: stepping through state succeeded with error <nil>
I200325 05:20:31.679150 20710 jobs/registry.go:764  [n1] job 540813651369295873: stepping through state succeeded with error <nil>
I200325 05:20:31.686497 20684 jobs/registry.go:764  [n1] job 540813651370835969: stepping through state succeeded with error <nil>
I200325 05:20:31.785406 20684 jobs/registry.go:916  [n1] job 540813651370835969: status succeeded after adoption finished
I200325 05:20:31.788244 20710 jobs/registry.go:916  [n1] job 540813651369295873: status succeeded after adoption finished
I200325 05:20:31.794692 19675 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 3.278986299s, 5.36 MiB/s)
I200325 05:20:31.794865 20692 jobs/registry.go:916  [n1] job 540813651287015425: status succeeded after adoption finished
I200325 05:20:31.800715 19676 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 3.289556358s, 1.30 MiB/s)
I200325 05:20:31.810983 19679 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 3.299550087s, 2.36 MiB/s)
I200325 05:20:31.851036 20654 jobs/registry.go:764  [n1] job 540813651239927809: stepping through state succeeded with error <nil>
I200325 05:20:31.911386 20654 jobs/registry.go:916  [n1] job 540813651239927809: status succeeded after adoption finished
I200325 05:20:31.921757 19677 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 3.410377868s, 0.44 MiB/s)
I200325 05:20:32.309210 21400 kv/kvserver/replica_command.go:397  [n1,s1,r56/1:/Table/57/{1-2}] initiating a split of this range at key /Table/57/1/0/1 [r78] (manual)
I200325 05:20:32.552812 20620 jobs/registry.go:764  [n1] job 540813651238649857: stepping through state succeeded with error <nil>
I200325 05:20:32.563293 20620 jobs/registry.go:916  [n1] job 540813651238649857: status succeeded after adoption finished
I200325 05:20:32.563538 19680 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 4.05204719s, 7.84 MiB/s)
I200325 05:20:32.588366 20933 kv/kvserver/replica_command.go:397  [n1,s1,r33/1:/Table/55/{1-2}] initiating a split of this range at key /Table/55/1/0/1/-3001/1 [r79] (manual)
I200325 05:20:33.193923 20623 jobs/registry.go:764  [n1] job 540813651230883841: stepping through state succeeded with error <nil>
I200325 05:20:33.206688 20623 jobs/registry.go:916  [n1] job 540813651230883841: status succeeded after adoption finished
I200325 05:20:33.206948 19681 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 4.696117346s, 4.63 MiB/s)
I200325 05:20:33.207185 19184 util/stop/stopper.go:539  quiescing
    --- FAIL: TestAllRegisteredImportFixture/tpcc (5.23s)
        all_test.go:95: sql: no rows in result set
            importing table district
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1837343&tab=buildLog) on [master@a8cfc851114655807f379a8a41f3ae3863b19e2d](https://github.com/cockroachdb/cockroach/commits/a8cfc851114655807f379a8a41f3ae3863b19e2d):

```
I200330 14:49:02.192596 20532 jobs/registry.go:952  [n1] job 542341002257858561: status succeeded after adoption finished
I200330 14:49:02.240387 20439 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 3.194019819s, 2.43 MiB/s)
I200330 14:49:02.697311 20579 jobs/registry.go:797  [n1] job 542341002275848193: stepping through state succeeded with error <nil>
I200330 14:49:02.763333 20579 jobs/registry.go:952  [n1] job 542341002275848193: status succeeded after adoption finished
I200330 14:49:02.789906 20565 jobs/registry.go:797  [n1] job 542341002320969729: stepping through state succeeded with error <nil>
I200330 14:49:02.807809 20565 jobs/registry.go:952  [n1] job 542341002320969729: status succeeded after adoption finished
I200330 14:49:02.827136 20437 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 3.780637756s, 0.40 MiB/s)
I200330 14:49:02.876381 20538 jobs/registry.go:797  [n1] job 542341002323230721: stepping through state succeeded with error <nil>
I200330 14:49:02.907984 20538 jobs/registry.go:952  [n1] job 542341002323230721: status succeeded after adoption finished
I200330 14:49:02.920595 20436 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 3.876937224s, 1.11 MiB/s)
I200330 14:49:03.014908 21268 kv/kvserver/replica_command.go:397  [n1,s1,r49/1:/Table/61/{1-2}] initiating a split of this range at key /Table/61/1/0/1 [r69] (manual)
I200330 14:49:03.746977 20771 jobs/registry.go:797  [n1] job 542341002620305409: stepping through state succeeded with error <nil>
I200330 14:49:03.758309 20771 jobs/registry.go:952  [n1] job 542341002620305409: status succeeded after adoption finished
I200330 14:49:03.758583 20440 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 4.7025978s, 6.76 MiB/s)
I200330 14:49:03.986336 20887 kv/kvserver/replica_command.go:397  [n1,s1,r36/1:/Table/53/{1-2}] initiating a split of this range at key /Table/53/1/0/1/-3001/1 [r70] (manual)
I200330 14:49:04.727030 20498 jobs/registry.go:797  [n1] job 542341002254352385: stepping through state succeeded with error <nil>
I200330 14:49:04.735020 20498 jobs/registry.go:952  [n1] job 542341002254352385: status succeeded after adoption finished
I200330 14:49:04.735243 20441 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 5.692464758s, 3.82 MiB/s)
I200330 14:49:04.735459 18508 util/stop/stopper.go:539  quiescing
W200330 14:49:04.735598 22428 kv/kvserver/intentresolver/intent_resolver.go:732  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
I200330 14:49:04.749205 19226 kv/kvserver/queue.go:578  [n1,s1] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200330 14:49:04.749256 19226 kv/kvserver/queue.go:578  [n1,s1] rate limited in MaybeAdd (merge): node unavailable; try another peer
I200330 14:49:04.749276 19226 kv/kvserver/queue.go:578  [n1,s1] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200330 14:49:04.749286 19226 kv/kvserver/queue.go:578  [n1,s1] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
I200330 14:49:04.749295 19226 kv/kvserver/queue.go:578  [n1,s1] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200330 14:49:04.749306 19226 kv/kvserver/queue.go:578  [n1,s1] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200330 14:49:04.749315 19226 kv/kvserver/queue.go:578  [n1,s1] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200330 14:49:04.749326 19226 kv/kvserver/queue.go:578  [n1,s1] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (6.18s)
        all_test.go:95: sql: no rows in result set
            importing table customer
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1845023&tab=buildLog) on [master@51e8960053a8224856234ffc1fb610c3843f7d7d](https://github.com/cockroachdb/cockroach/commits/51e8960053a8224856234ffc1fb610c3843f7d7d):

```
I200402 02:13:39.577639 18339 jobs/registry.go:797  [n1] job 543041840650846209: stepping through state succeeded with error <nil>
I200402 02:13:39.646174 18392 jobs/registry.go:797  [n1] job 543041840719888385: stepping through state succeeded with error <nil>
I200402 02:13:39.662335 18339 jobs/registry.go:952  [n1] job 543041840650846209: status succeeded after adoption finished
I200402 02:13:39.966920 18351 kv/kvserver/replica_command.go:397  [n1,s1,r80/1:/Table/{59/1-60}] initiating a split of this range at key /Table/59/2 [r81] (manual)
I200402 02:13:40.012132 18392 jobs/registry.go:952  [n1] job 543041840719888385: status succeeded after adoption finished
I200402 02:13:40.063731 18277 ccl/workloadccl/fixture.go:446  imported 114 KiB in new_order table (9000 rows, 0 index entries, took 2.110495518s, 0.05 MiB/s)
I200402 02:13:40.310069 17378 gossip/gossip.go:1527  [n1] node has connected to cluster via gossip
I200402 02:13:40.310256 17378 kv/kvserver/stores.go:266  [n1] wrote 0 node addresses to persistent storage
I200402 02:13:40.479944 19024 kv/kvserver/replica_command.go:397  [n1,s1,r52/1:/Table/60/{1-2}] initiating a split of this range at key /Table/60/1/0/1/1 [r82] (manual)
I200402 02:13:41.241571 18447 jobs/registry.go:797  [n1] job 543041840784605185: stepping through state succeeded with error <nil>
I200402 02:13:41.321669 18447 jobs/registry.go:952  [n1] job 543041840784605185: status succeeded after adoption finished
I200402 02:13:41.331792 18274 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 3.383981988s, 5.19 MiB/s)
I200402 02:13:41.862238 18362 jobs/registry.go:797  [n1] job 543041840709238785: stepping through state succeeded with error <nil>
I200402 02:13:41.912348 18362 jobs/registry.go:952  [n1] job 543041840709238785: status succeeded after adoption finished
I200402 02:13:41.913904 18275 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 3.961080491s, 1.08 MiB/s)
I200402 02:13:41.980830 18402 jobs/registry.go:797  [n1] job 543041840717922305: stepping through state succeeded with error <nil>
I200402 02:13:42.005953 18402 jobs/registry.go:952  [n1] job 543041840717922305: status succeeded after adoption finished
I200402 02:13:42.006616 18278 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 4.053181787s, 1.92 MiB/s)
I200402 02:13:42.268469 19573 kv/kvserver/replica_command.go:397  [n1,s1,r80/1:/Table/59/{1-2}] initiating a split of this range at key /Table/59/1/0/1 [r83] (manual)
I200402 02:13:42.592466 18580 kv/kvserver/replica_command.go:397  [n1,s1,r33/1:/Table/53/{1-2}] initiating a split of this range at key /Table/53/1/0/1/-3001/1 [r84] (manual)
I200402 02:13:42.758612 18351 jobs/registry.go:797  [n1] job 543041840771956737: stepping through state succeeded with error <nil>
I200402 02:13:42.773583 18351 jobs/registry.go:952  [n1] job 543041840771956737: status succeeded after adoption finished
I200402 02:13:42.773847 18279 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 4.803912052s, 6.61 MiB/s)
I200402 02:13:43.462329 18313 jobs/registry.go:797  [n1] job 543041840596844545: stepping through state succeeded with error <nil>
I200402 02:13:43.478757 18313 jobs/registry.go:952  [n1] job 543041840596844545: status succeeded after adoption finished
I200402 02:13:43.479399 18280 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 5.532053511s, 3.93 MiB/s)
I200402 02:13:43.479574 17310 util/stop/stopper.go:539  quiescing
W200402 02:13:43.479715 20340 kv/kvserver/intentresolver/intent_resolver.go:732  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (5.75s)
        all_test.go:95: sql: no rows in result set
            importing table order
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1858691&tab=buildLog) on [master@101e52af4f2bf2e36059621097e32ce48c0e2efa](https://github.com/cockroachdb/cockroach/commits/101e52af4f2bf2e36059621097e32ce48c0e2efa):

```
I200407 12:06:43.838435 18318 jobs/registry.go:952  [n1] job 544574019395059713: status succeeded after adoption finished
I200407 12:06:43.857535 19106 kv/kvserver/split_queue.go:169  [n1,split,s1,r71/1:/Table/60{-/1}] split saw concurrent descriptor modification; maybe retrying
I200407 12:06:43.884815 17743 ccl/workloadccl/fixture.go:446  imported 53 B in warehouse table (1 rows, 0 index entries, took 2.122888085s, 0.00 MiB/s)
I200407 12:06:44.297808 18189 jobs/registry.go:797  [n1] job 544574019322740737: stepping through state succeeded with error <nil>
I200407 12:06:44.361517 18447 kv/kvserver/replica_command.go:397  [n1,s1,r42/1:/Table/53/{1-2}] initiating a split of this range at key /Table/53/1/0/1/1 [r80] (manual)
I200407 12:06:44.476461 18189 jobs/registry.go:952  [n1] job 544574019322740737: status succeeded after adoption finished
I200407 12:06:44.492182 18084 ccl/workloadccl/fixture.go:446  imported 114 KiB in new_order table (9000 rows, 0 index entries, took 2.728273979s, 0.04 MiB/s)
I200407 12:06:44.602599 18016 jobs/registry.go:797  [n1] job 544574019293020161: stepping through state succeeded with error <nil>
I200407 12:06:44.678980 18016 jobs/registry.go:952  [n1] job 544574019293020161: status succeeded after adoption finished
I200407 12:06:44.689896 18083 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 2.926125138s, 0.51 MiB/s)
I200407 12:06:44.732358 18113 jobs/registry.go:797  [n1] job 544574019291119617: stepping through state succeeded with error <nil>
W200407 12:06:44.736476 19312 kv/kvserver/intentresolver/intent_resolver.go:441  failed to push during intent resolution: failed to push meta={id=a44c2a85 key=/Table/SystemConfigSpan/Start pri=0.00633830 epo=0 ts=1586261204.727108089,0 min=1586261204.727108089,0 seq=1} lock=true stat=PENDING rts=0,0 wto=false max=0,0
I200407 12:06:44.738737 18133 jobs/registry.go:797  [n1] job 544574019265232897: stepping through state succeeded with error <nil>
I200407 12:06:44.760935 18133 jobs/registry.go:952  [n1] job 544574019265232897: status succeeded after adoption finished
I200407 12:06:44.852934 18113 jobs/registry.go:952  [n1] job 544574019291119617: status succeeded after adoption finished
I200407 12:06:44.857729 18085 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 3.09370013s, 2.51 MiB/s)
I200407 12:06:44.925302 18106 jobs/registry.go:797  [n1] job 544574019271786497: stepping through state succeeded with error <nil>
I200407 12:06:44.941862 18106 jobs/registry.go:952  [n1] job 544574019271786497: status succeeded after adoption finished
I200407 12:06:44.957062 18082 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 3.193439318s, 1.34 MiB/s)
I200407 12:06:44.985417 18548 kv/kvserver/replica_command.go:397  [n1,s1,r44/1:/Table/55/{1-2}] initiating a split of this range at key /Table/55/1/0/1 [r81] (manual)
I200407 12:06:45.542128 18138 jobs/registry.go:797  [n1] job 544574019266674689: stepping through state succeeded with error <nil>
I200407 12:06:45.551852 18138 jobs/registry.go:952  [n1] job 544574019266674689: status succeeded after adoption finished
I200407 12:06:45.552113 18086 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 3.787988768s, 8.39 MiB/s)
I200407 12:06:45.829285 18781 kv/kvserver/replica_command.go:397  [n1,s1,r51/1:/Table/59/{1-2}] initiating a split of this range at key /Table/59/1/0/1/-3001/1 [r82] (manual)
I200407 12:06:46.334630 18244 jobs/registry.go:797  [n1] job 544574019227287553: stepping through state succeeded with error <nil>
I200407 12:06:46.345830 18244 jobs/registry.go:952  [n1] job 544574019227287553: status succeeded after adoption finished
I200407 12:06:46.346194 18087 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 4.585369827s, 4.74 MiB/s)
I200407 12:06:46.346448 16817 util/stop/stopper.go:539  quiescing
    --- FAIL: TestAllRegisteredImportFixture/tpcc (4.75s)
        all_test.go:95: sql: no rows in result set
            importing table customer
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1865040&tab=buildLog) on [master@4bdcec342248a1ee90f2d36fe4650ada4f6e4b8c](https://github.com/cockroachdb/cockroach/commits/4bdcec342248a1ee90f2d36fe4650ada4f6e4b8c):

```
I200409 16:38:13.871050 17659 gossip/gossip.go:1527  [n1] node has connected to cluster via gossip
I200409 16:38:13.871223 17659 kv/kvserver/stores.go:266  [n1] wrote 0 node addresses to persistent storage
I200409 16:38:14.215770 18743 jobs/registry.go:797  [n1] job 545193629873405953: stepping through state succeeded with error <nil>
I200409 16:38:14.531816 18743 jobs/registry.go:952  [n1] job 545193629873405953: status succeeded after adoption finished
I200409 16:38:14.584715 17953 ccl/workloadccl/fixture.go:446  imported 114 KiB in new_order table (9000 rows, 0 index entries, took 2.702349335s, 0.04 MiB/s)
I200409 16:38:15.482148 18653 jobs/registry.go:797  [n1] job 545193629859151873: stepping through state succeeded with error <nil>
I200409 16:38:15.507068 18691 jobs/registry.go:797  [n1] job 545193629881860097: stepping through state succeeded with error <nil>
I200409 16:38:15.521974 18691 jobs/registry.go:952  [n1] job 545193629881860097: status succeeded after adoption finished
I200409 16:38:15.525590 18653 jobs/registry.go:952  [n1] job 545193629859151873: status succeeded after adoption finished
I200409 16:38:15.529183 17952 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 3.646928671s, 0.41 MiB/s)
I200409 16:38:15.529392 17951 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 3.647257388s, 1.18 MiB/s)
I200409 16:38:15.601404 19193 kv/kvserver/replica_command.go:397  [n1,s1,r46/1:/Table/57/{1-2}] initiating a split of this range at key /Table/57/1/0/1/1 [r78] (manual)
I200409 16:38:15.893645 18680 jobs/registry.go:797  [n1] job 545193629874094081: stepping through state succeeded with error <nil>
I200409 16:38:16.011071 18709 jobs/registry.go:797  [n1] job 545193629859053569: stepping through state succeeded with error <nil>
I200409 16:38:16.019737 18680 jobs/registry.go:952  [n1] job 545193629874094081: status succeeded after adoption finished
I200409 16:38:16.056766 18594 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 4.16069689s, 1.87 MiB/s)
I200409 16:38:16.105342 19537 kv/kvserver/replica_command.go:397  [n1,s1,r75/1:/Table/59/{1-2}] initiating a split of this range at key /Table/59/1/0/1 [r79] (manual)
I200409 16:38:16.106489 18709 jobs/registry.go:952  [n1] job 545193629859053569: status succeeded after adoption finished
I200409 16:38:16.144680 17950 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 4.26261942s, 4.12 MiB/s)
I200409 16:38:17.139273 18663 jobs/registry.go:797  [n1] job 545193629869899777: stepping through state succeeded with error <nil>
I200409 16:38:17.180653 18663 jobs/registry.go:952  [n1] job 545193629869899777: status succeeded after adoption finished
I200409 16:38:17.180966 18595 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 5.284223931s, 6.01 MiB/s)
I200409 16:38:17.720852 19590 kv/kvserver/replica_command.go:397  [n1,s1,r72/1:/Table/55/{1-2}] initiating a split of this range at key /Table/55/1/0/1/-3001/1 [r80] (manual)
I200409 16:38:19.011340 18636 jobs/registry.go:797  [n1] job 545193629858562049: stepping through state succeeded with error <nil>
I200409 16:38:19.025529 18636 jobs/registry.go:952  [n1] job 545193629858562049: status succeeded after adoption finished
I200409 16:38:19.025931 18596 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 7.144282103s, 3.04 MiB/s)
I200409 16:38:19.026381 17384 util/stop/stopper.go:539  quiescing
W200409 16:38:19.026573 20748 kv/kvserver/intentresolver/intent_resolver.go:745  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (7.52s)
        all_test.go:95: sql: no rows in result set
            importing table district
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1876648&tab=buildLog) on [master@18b0247935ce5e011599cd5a93ef3c6679ebb404](https://github.com/cockroachdb/cockroach/commits/18b0247935ce5e011599cd5a93ef3c6679ebb404):

```
I200414 23:44:39.310700 18378 ccl/workloadccl/fixture.go:446  imported 114 KiB in new_order table (9000 rows, 0 index entries, took 930.979835ms, 0.12 MiB/s)
I200414 23:44:39.328578 18488 kv/kvserver/replica_command.go:397  [n1,s1,r68/1:/Table/5{7/1-8}] initiating a split of this range at key /Table/57/2 [r69] (manual)
I200414 23:44:39.334451 18502 kv/kvserver/replica_command.go:397  [n1,s1,r67/1:/Table/5{8/1-9}] initiating a split of this range at key /Table/58/2 [r70] (manual)
I200414 23:44:39.987736 17477 gossip/gossip.go:1538  [n1] node has connected to cluster via gossip
I200414 23:44:39.987929 17477 kv/kvserver/stores.go:255  [n1] wrote 0 node addresses to persistent storage
I200414 23:44:40.420744 18514 jobs/registry.go:797  [n1] job 546693049312215041: stepping through state succeeded with error <nil>
I200414 23:44:40.529774 18514 jobs/registry.go:952  [n1] job 546693049312215041: status succeeded after adoption finished
I200414 23:44:40.550043 18379 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 2.17035522s, 3.58 MiB/s)
I200414 23:44:40.710379 18304 jobs/registry.go:797  [n1] job 546693049390202881: stepping through state succeeded with error <nil>
I200414 23:44:40.792138 18304 jobs/registry.go:952  [n1] job 546693049390202881: status succeeded after adoption finished
I200414 23:44:40.810168 18377 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 2.430724838s, 0.62 MiB/s)
I200414 23:44:41.257593 18531 jobs/registry.go:797  [n1] job 546693049419431937: stepping through state succeeded with error <nil>
I200414 23:44:41.346007 19513 kv/kvserver/replica_command.go:397  [n1,s1,r68/1:/Table/57/{1-2}] initiating a split of this range at key /Table/57/1/0/1/1 [r71] (manual)
I200414 23:44:41.369185 18531 jobs/registry.go:952  [n1] job 546693049419431937: status succeeded after adoption finished
I200414 23:44:41.381582 18376 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 3.002258826s, 1.43 MiB/s)
I200414 23:44:41.539086 19144 kv/kvserver/replica_command.go:397  [n1,s1,r52/1:/Table/54/{1-2}] initiating a split of this range at key /Table/54/1/0/1 [r72] (manual)
I200414 23:44:41.755269 18488 jobs/registry.go:797  [n1] job 546693049316179969: stepping through state succeeded with error <nil>
I200414 23:44:41.816712 18488 jobs/registry.go:952  [n1] job 546693049316179969: status succeeded after adoption finished
I200414 23:44:41.830120 18375 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 3.450922468s, 5.09 MiB/s)
I200414 23:44:42.276799 18517 jobs/registry.go:797  [n1] job 546693049313918977: stepping through state succeeded with error <nil>
I200414 23:44:42.312026 18517 jobs/registry.go:952  [n1] job 546693049313918977: status succeeded after adoption finished
I200414 23:44:42.395600 19943 kv/kvserver/replica_consistency.go:255  [n1,consistencyChecker,s1,r4/1:/System{/tsd-tse}] triggering stats recomputation to resolve delta of {ContainsEstimates:1448 LastUpdateNanos:1586907878254535832 IntentAge:0 GCBytesAge:0 LiveBytes:-35490 LiveCount:-697 KeyBytes:-33971 KeyCount:-697 ValBytes:-1519 ValCount:-697 IntentBytes:0 IntentCount:0 SysBytes:0 SysCount:0}
I200414 23:44:43.006557 19471 kv/kvserver/replica_command.go:397  [n1,s1,r67/1:/Table/58/{1-2}] initiating a split of this range at key /Table/58/1/0/1/-3001/1 [r73] (manual)
I200414 23:44:43.671308 18502 jobs/registry.go:797  [n1] job 546693049306316801: stepping through state succeeded with error <nil>
I200414 23:44:43.681571 18502 jobs/registry.go:952  [n1] job 546693049306316801: status succeeded after adoption finished
I200414 23:44:43.681839 18381 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 5.302881696s, 4.10 MiB/s)
I200414 23:44:43.682091 17098 util/stop/stopper.go:539  quiescing
W200414 23:44:43.682224 20126 kv/kvserver/intentresolver/intent_resolver.go:745  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (5.58s)
        all_test.go:95: sql: no rows in result set
            importing table stock
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1877058&tab=buildLog) on [master@8c928666de3f90cf17d81d70a3ee759c39e2ce27](https://github.com/cockroachdb/cockroach/commits/8c928666de3f90cf17d81d70a3ee759c39e2ce27):

```
I200415 05:26:20.633705 17467 kv/kvserver/stores.go:255  [n1] wrote 0 node addresses to persistent storage
I200415 05:26:20.688565 19353 kv/kvserver/split_queue.go:169  [n1,split,s1,r62/1:/Table/59{-/1}] split saw concurrent descriptor modification; maybe retrying
I200415 05:26:20.728451 18595 jobs/registry.go:958  [n1] job 546760224090587137: status succeeded after adoption finished
I200415 05:26:20.738490 18288 ccl/workloadccl/fixture.go:446  imported 53 B in warehouse table (1 rows, 0 index entries, took 2.230961008s, 0.00 MiB/s)
I200415 05:26:20.920393 18161 kv/kvserver/replica_command.go:397  [n1,s1,r66/1:/Table/{59/1-60}] initiating a split of this range at key /Table/59/2 [r70] (manual)
I200415 05:26:22.918447 19630 kv/kvserver/replica_consistency.go:255  [n1,consistencyChecker,s1,r4/1:/System{/tsd-tse}] triggering stats recomputation to resolve delta of {ContainsEstimates:1448 LastUpdateNanos:1586928378414647168 IntentAge:0 GCBytesAge:0 LiveBytes:-35490 LiveCount:-697 KeyBytes:-33971 KeyCount:-697 ValBytes:-1519 ValCount:-697 IntentBytes:0 IntentCount:0 SysBytes:0 SysCount:0}
I200415 05:26:23.027489 18767 kv/kvserver/replica_command.go:397  [n1,s1,r40/1:/Table/60/{1-2}] initiating a split of this range at key /Table/60/1/0/1/1 [r71] (manual)
I200415 05:26:23.154974 18161 jobs/registry.go:803  [n1] job 546760224066437121: stepping through state succeeded with error <nil>
I200415 05:26:23.226438 18161 jobs/registry.go:958  [n1] job 546760224066437121: status succeeded after adoption finished
I200415 05:26:23.238032 18420 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 4.730077255s, 0.32 MiB/s)
I200415 05:26:23.429592 18799 kv/kvserver/replica_command.go:397  [n1,s1,r34/1:/Table/53/{1-2}] initiating a split of this range at key /Table/53/1/0/1 [r72] (manual)
I200415 05:26:23.475894 18489 jobs/registry.go:803  [n1] job 546760224087572481: stepping through state succeeded with error <nil>
I200415 05:26:23.541233 18489 jobs/registry.go:958  [n1] job 546760224087572481: status succeeded after adoption finished
I200415 05:26:23.551404 18422 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 5.04321907s, 1.54 MiB/s)
I200415 05:26:23.630848 17601 jobs/registry.go:803  [n1] job 546760224059392001: stepping through state succeeded with error <nil>
I200415 05:26:23.663353 17601 jobs/registry.go:958  [n1] job 546760224059392001: status succeeded after adoption finished
I200415 05:26:23.666577 18663 jobs/registry.go:803  [n1] job 546760224219496449: stepping through state succeeded with error <nil>
I200415 05:26:23.714408 18663 jobs/registry.go:958  [n1] job 546760224219496449: status succeeded after adoption finished
I200415 05:26:23.733528 18418 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 5.22584735s, 3.36 MiB/s)
I200415 05:26:24.466036 18093 jobs/registry.go:803  [n1] job 546760224064241665: stepping through state succeeded with error <nil>
I200415 05:26:24.479561 18093 jobs/registry.go:958  [n1] job 546760224064241665: status succeeded after adoption finished
I200415 05:26:24.480454 18423 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 5.972159031s, 5.32 MiB/s)
I200415 05:26:24.764737 18743 kv/kvserver/replica_command.go:397  [n1,s1,r35/1:/Table/57/{1-2}] initiating a split of this range at key /Table/57/1/0/1/-3001/1 [r73] (manual)
I200415 05:26:25.263636 18500 jobs/registry.go:803  [n1] job 546760224078888961: stepping through state succeeded with error <nil>
I200415 05:26:25.273339 18500 jobs/registry.go:958  [n1] job 546760224078888961: status succeeded after adoption finished
I200415 05:26:25.273553 18424 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 6.766102368s, 3.21 MiB/s)
I200415 05:26:25.273793 17392 util/stop/stopper.go:539  quiescing
W200415 05:26:25.273976 20169 kv/kvserver/intentresolver/intent_resolver.go:745  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (6.93s)
        all_test.go:95: sql: no rows in result set
            importing table history
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1879399&tab=buildLog) on [master@de48c6d13431061a6227697a917e574464b80983](https://github.com/cockroachdb/cockroach/commits/de48c6d13431061a6227697a917e574464b80983):

```
I200416 05:51:59.428379 18161 kv/kvserver/replica_command.go:397  [n1,s1,r65/1:/Table/5{5/1-6}] initiating a split of this range at key /Table/55/2 [r66] (manual)
I200416 05:51:59.436832 18014 kv/kvserver/replica_command.go:397  [n1,s1,r63/1:/Table/5{4/2-5}] initiating a split of this range at key /Table/54/3 [r67] (manual)
I200416 05:51:59.448809 18174 kv/kvserver/replica_command.go:397  [n1,s1,r62/1:/Table/5{8/1-9}] initiating a split of this range at key /Table/58/2 [r68] (manual)
I200416 05:52:00.259805 17142 gossip/gossip.go:1538  [n1] node has connected to cluster via gossip
I200416 05:52:00.261485 17142 kv/kvserver/stores.go:255  [n1] wrote 0 node addresses to persistent storage
I200416 05:52:00.513863 18202 jobs/registry.go:803  [n1] job 547048385180401665: stepping through state succeeded with error <nil>
I200416 05:52:00.653053 18202 jobs/registry.go:958  [n1] job 547048385180401665: status succeeded after adoption finished
I200416 05:52:00.661548 18107 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 2.36886384s, 3.28 MiB/s)
I200416 05:52:01.211337 18171 jobs/registry.go:803  [n1] job 547048385147961345: stepping through state succeeded with error <nil>
I200416 05:52:01.248610 19287 kv/kvserver/replica_consistency.go:255  [n1,consistencyChecker,s1,r4/1:/System{/tsd-tse}] triggering stats recomputation to resolve delta of {ContainsEstimates:1448 LastUpdateNanos:1587016318191740332 IntentAge:0 GCBytesAge:0 LiveBytes:-35490 LiveCount:-697 KeyBytes:-33971 KeyCount:-697 ValBytes:-1519 ValCount:-697 IntentBytes:0 IntentCount:0 SysBytes:0 SysCount:0}
I200416 05:52:01.303339 18171 jobs/registry.go:958  [n1] job 547048385147961345: status succeeded after adoption finished
I200416 05:52:01.816710 18014 jobs/registry.go:803  [n1] job 547048385159299073: stepping through state succeeded with error <nil>
I200416 05:52:01.914149 18014 jobs/registry.go:958  [n1] job 547048385159299073: status succeeded after adoption finished
I200416 05:52:01.924818 18104 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 3.632442227s, 1.18 MiB/s)
I200416 05:52:01.961892 19048 kv/kvserver/replica_command.go:397  [n1,s1,r62/1:/Table/58/{1-2}] initiating a split of this range at key /Table/58/1/0/1/1 [r69] (manual)
I200416 05:52:02.258809 18174 jobs/registry.go:803  [n1] job 547048385204944897: stepping through state succeeded with error <nil>
I200416 05:52:02.376920 19072 kv/kvserver/replica_command.go:397  [n1,s1,r65/1:/Table/55/{1-2}] initiating a split of this range at key /Table/55/1/0/1 [r70] (manual)
I200416 05:52:02.421458 18174 jobs/registry.go:958  [n1] job 547048385204944897: status succeeded after adoption finished
I200416 05:52:02.427963 18103 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 4.135710972s, 4.25 MiB/s)
I200416 05:52:02.724240 18161 jobs/registry.go:803  [n1] job 547048385212841985: stepping through state succeeded with error <nil>
I200416 05:52:02.735126 18161 jobs/registry.go:958  [n1] job 547048385212841985: status succeeded after adoption finished
I200416 05:52:02.735279 18108 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 4.442520066s, 7.15 MiB/s)
I200416 05:52:02.876467 18757 kv/kvserver/replica_command.go:397  [n1,s1,r55/1:/Table/53/{1-2}] initiating a split of this range at key /Table/53/1/0/1/-3001/1 [r71] (manual)
I200416 05:52:03.244868 18126 jobs/registry.go:803  [n1] job 547048385109327873: stepping through state succeeded with error <nil>
I200416 05:52:03.253945 18126 jobs/registry.go:958  [n1] job 547048385109327873: status succeeded after adoption finished
I200416 05:52:03.254112 18109 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 4.962261202s, 4.38 MiB/s)
I200416 05:52:03.254288 17106 util/stop/stopper.go:539  quiescing
W200416 05:52:03.254422 19729 kv/kvserver/intentresolver/intent_resolver.go:745  failed to gc transaction record: could not GC completed transaction anchored at /Local/Range/Table/55/1/RangeDescriptor: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (5.14s)
        all_test.go:95: sql: no rows in result set
            importing table order
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1881791&tab=buildLog) on [master@a63a670b1a591be2ed38af7f98c1f85e32546596](https://github.com/cockroachdb/cockroach/commits/a63a670b1a591be2ed38af7f98c1f85e32546596):

```
I200417 05:21:06.909844 17284 gossip/gossip.go:1538  [n1] node has connected to cluster via gossip
I200417 05:21:06.910100 17284 kv/kvserver/stores.go:255  [n1] wrote 0 node addresses to persistent storage
W200417 05:21:07.877069 17352 kv/kvserver/store_raft.go:508  [n1,s1,r64/1:/Table/55/{1-2}] handle raft ready: 0.7s [applied=1, batches=1, state_assertions=0]
I200417 05:21:08.230900 18381 jobs/registry.go:803  [n1] job 547325425835704321: stepping through state succeeded with error <nil>
I200417 05:21:08.376982 18381 jobs/registry.go:958  [n1] job 547325425835704321: status succeeded after adoption finished
I200417 05:21:08.605410 18093 jobs/registry.go:803  [n1] job 547325425746051073: stepping through state succeeded with error <nil>
I200417 05:21:08.677647 18976 kv/kvserver/replica_command.go:397  [n1,s1,r69/1:/Table/58/{1-2}] initiating a split of this range at key /Table/58/1/0/1/1 [r73] (manual)
I200417 05:21:08.730324 18079 jobs/registry.go:803  [n1] job 547325425724981249: stepping through state succeeded with error <nil>
I200417 05:21:08.755005 18093 jobs/registry.go:958  [n1] job 547325425746051073: status succeeded after adoption finished
I200417 05:21:08.760215 18079 jobs/registry.go:958  [n1] job 547325425724981249: status succeeded after adoption finished
I200417 05:21:08.760560 18063 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 4.382597082s, 0.34 MiB/s)
I200417 05:21:08.797539 18062 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 4.419318127s, 0.97 MiB/s)
I200417 05:21:09.105620 18320 jobs/registry.go:803  [n1] job 547325425733435393: stepping through state succeeded with error <nil>
I200417 05:21:09.145725 18320 jobs/registry.go:958  [n1] job 547325425733435393: status succeeded after adoption finished
I200417 05:21:09.159181 18061 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 4.781165076s, 3.67 MiB/s)
I200417 05:21:09.299528 18776 kv/kvserver/replica_command.go:397  [n1,s1,r50/1:/Table/61/{1-2}] initiating a split of this range at key /Table/61/1/0/1 [r74] (manual)
I200417 05:21:09.379676 18338 jobs/registry.go:803  [n1] job 547325425752768513: stepping through state succeeded with error <nil>
I200417 05:21:09.458107 18338 jobs/registry.go:958  [n1] job 547325425752768513: status succeeded after adoption finished
I200417 05:21:09.484613 18065 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 5.106362238s, 1.52 MiB/s)
I200417 05:21:10.175869 18289 jobs/registry.go:803  [n1] job 547325425752899585: stepping through state succeeded with error <nil>
I200417 05:21:10.185218 18289 jobs/registry.go:958  [n1] job 547325425752899585: status succeeded after adoption finished
I200417 05:21:10.185541 18258 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 5.807174347s, 5.47 MiB/s)
I200417 05:21:10.913040 18841 kv/kvserver/replica_command.go:397  [n1,s1,r38/1:/Table/60/{1-2}] initiating a split of this range at key /Table/60/1/0/1/-3001/1 [r75] (manual)
I200417 05:21:11.439846 18438 jobs/registry.go:803  [n1] job 547325425779212289: stepping through state succeeded with error <nil>
I200417 05:21:11.448547 18438 jobs/registry.go:958  [n1] job 547325425779212289: status succeeded after adoption finished
I200417 05:21:11.448777 18259 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 7.070912283s, 3.07 MiB/s)
I200417 05:21:11.448995 17200 util/stop/stopper.go:539  quiescing
W200417 05:21:11.449097 19772 kv/kvserver/intentresolver/intent_resolver.go:745  failed to gc transaction record: could not GC completed transaction anchored at /Local/Range/Table/60/1/RangeDescriptor: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (7.20s)
        all_test.go:95: sql: no rows in result set
            importing table new_order
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1883745&tab=buildLog) on [master@b5b4a9a55f1122ef9c82b968aa5c8cc137c7e281](https://github.com/cockroachdb/cockroach/commits/b5b4a9a55f1122ef9c82b968aa5c8cc137c7e281):

```
I200418 05:35:56.280767 18221 jobs/registry.go:803  [n1] job 547611461353340929: stepping through state succeeded with error <nil>
I200418 05:35:56.314713 18221 jobs/registry.go:958  [n1] job 547611461353340929: status succeeded after adoption finished
I200418 05:35:56.444147 17135 kv/kvserver/queue.go:578  [n1,s1] rate limited in MaybeAdd (split): throttled on async limiting semaphore
I200418 05:35:56.568136 18166 kv/kvserver/replica_command.go:397  [n1,s1,r68/1:/Table/{59/1-60}] initiating a split of this range at key /Table/59/2 [r70] (manual)
I200418 05:35:56.751370 18234 jobs/registry.go:958  [n1] job 547611461350162433: status succeeded after adoption finished
I200418 05:35:56.754410 18119 ccl/workloadccl/fixture.go:446  imported 114 KiB in new_order table (9000 rows, 0 index entries, took 1.241461577s, 0.09 MiB/s)
I200418 05:35:57.067892 17202 gossip/gossip.go:1538  [n1] node has connected to cluster via gossip
I200418 05:35:57.067995 17202 kv/kvserver/stores.go:255  [n1] wrote 0 node addresses to persistent storage
I200418 05:35:57.093565 17949 jobs/registry.go:803  [n1] job 547611461343051777: stepping through state succeeded with error <nil>
I200418 05:35:57.221731 17949 jobs/registry.go:958  [n1] job 547611461343051777: status succeeded after adoption finished
I200418 05:35:57.230898 18117 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 1.718103552s, 2.50 MiB/s)
I200418 05:35:57.716119 18633 kv/kvserver/replica_command.go:397  [n1,s1,r54/1:/Table/61/{1-2}] initiating a split of this range at key /Table/61/1/0/1 [r71] (manual)
I200418 05:35:57.742677 17952 jobs/registry.go:803  [n1] job 547611461364088833: stepping through state succeeded with error <nil>
I200418 05:35:57.832454 17952 jobs/registry.go:958  [n1] job 547611461364088833: status succeeded after adoption finished
I200418 05:35:57.849265 18120 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 2.336221345s, 3.33 MiB/s)
I200418 05:35:57.893106 19377 kv/kvserver/replica_command.go:397  [n1,s1,r68/1:/Table/59/{1-2}] initiating a split of this range at key /Table/59/1/0/1/1 [r72] (manual)
I200418 05:35:58.351908 18166 jobs/registry.go:803  [n1] job 547611461363335169: stepping through state succeeded with error <nil>
I200418 05:35:58.439415 18166 jobs/registry.go:958  [n1] job 547611461363335169: status succeeded after adoption finished
I200418 05:35:58.448079 18116 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 2.935370763s, 5.98 MiB/s)
I200418 05:35:58.941433 18271 jobs/registry.go:803  [n1] job 547611461338726401: stepping through state succeeded with error <nil>
I200418 05:35:58.951898 18271 jobs/registry.go:958  [n1] job 547611461338726401: status succeeded after adoption finished
I200418 05:35:58.952104 18121 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 3.43897024s, 9.24 MiB/s)
I200418 05:35:59.871548 18971 kv/kvserver/replica_command.go:397  [n1,s1,r42/1:/Table/55/{1-2}] initiating a split of this range at key /Table/55/1/0/1/-3001/1 [r73] (manual)
I200418 05:36:01.167972 18231 jobs/registry.go:803  [n1] job 547611461317033985: stepping through state succeeded with error <nil>
I200418 05:36:01.195709 18231 jobs/registry.go:958  [n1] job 547611461317033985: status succeeded after adoption finished
I200418 05:36:01.196024 18122 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 5.683581292s, 3.82 MiB/s)
I200418 05:36:01.196203 17111 util/stop/stopper.go:539  quiescing
W200418 05:36:01.196317 20225 kv/kvserver/intentresolver/intent_resolver.go:745  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (5.81s)
        all_test.go:95: sql: no rows in result set
            importing table order
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1884819&tab=buildLog) on [master@ce5d511368b741555344ea28efa40fb9facfa577](https://github.com/cockroachdb/cockroach/commits/ce5d511368b741555344ea28efa40fb9facfa577):

```
I200419 05:47:03.951961 18296 jobs/registry.go:958  [n1] job 547896760159535105: status succeeded after adoption finished
I200419 05:47:04.026465 18177 ccl/workloadccl/fixture.go:446  imported 1017 B in district table (10 rows, 0 index entries, took 2.227629341s, 0.00 MiB/s)
I200419 05:47:04.158429 17375 gossip/gossip.go:1538  [n1] node has connected to cluster via gossip
I200419 05:47:04.158569 17375 kv/kvserver/stores.go:255  [n1] wrote 0 node addresses to persistent storage
I200419 05:47:04.296558 18340 jobs/registry.go:958  [n1] job 547896760163336193: status succeeded after adoption finished
I200419 05:47:04.444187 18261 ccl/workloadccl/fixture.go:446  imported 114 KiB in new_order table (9000 rows, 0 index entries, took 2.645441602s, 0.04 MiB/s)
I200419 05:47:05.053616 18754 kv/kvserver/replica_command.go:397  [n1,s1,r42/1:/Table/59/{1-2}] initiating a split of this range at key /Table/59/1/0/1/1 [r79] (manual)
I200419 05:47:05.343915 18423 jobs/registry.go:803  [n1] job 547896760264196097: stepping through state succeeded with error <nil>
I200419 05:47:05.446681 18423 jobs/registry.go:958  [n1] job 547896760264196097: status succeeded after adoption finished
I200419 05:47:05.459964 18260 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 3.661402104s, 0.41 MiB/s)
I200419 05:47:05.531525 18392 jobs/registry.go:803  [n1] job 547896760218288129: stepping through state succeeded with error <nil>
I200419 05:47:05.591350 18392 jobs/registry.go:958  [n1] job 547896760218288129: status succeeded after adoption finished
I200419 05:47:05.607594 18258 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 3.808642786s, 4.61 MiB/s)
I200419 05:47:05.681510 18193 jobs/registry.go:803  [n1] job 547896760158093313: stepping through state succeeded with error <nil>
I200419 05:47:05.750320 18301 jobs/registry.go:803  [n1] job 547896760212652033: stepping through state succeeded with error <nil>
I200419 05:47:05.753595 18193 jobs/registry.go:958  [n1] job 547896760158093313: status succeeded after adoption finished
I200419 05:47:05.755800 18262 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 3.956939101s, 1.97 MiB/s)
I200419 05:47:05.803464 18301 jobs/registry.go:958  [n1] job 547896760212652033: status succeeded after adoption finished
I200419 05:47:05.813566 18259 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 4.015062285s, 1.07 MiB/s)
I200419 05:47:06.016230 18925 kv/kvserver/replica_command.go:397  [n1,s1,r55/1:/Table/61/{1-2}] initiating a split of this range at key /Table/61/1/0/1 [r80] (manual)
I200419 05:47:06.618625 18442 jobs/registry.go:803  [n1] job 547896760309284865: stepping through state succeeded with error <nil>
I200419 05:47:06.627752 18442 jobs/registry.go:958  [n1] job 547896760309284865: status succeeded after adoption finished
I200419 05:47:06.628013 18263 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 4.829060411s, 6.58 MiB/s)
I200419 05:47:06.922020 18994 kv/kvserver/replica_command.go:397  [n1,s1,r44/1:/Table/53/{1-2}] initiating a split of this range at key /Table/53/1/0/1/-3001/1 [r81] (manual)
I200419 05:47:07.357331 18291 jobs/registry.go:803  [n1] job 547896760105435137: stepping through state succeeded with error <nil>
I200419 05:47:07.365525 18291 jobs/registry.go:958  [n1] job 547896760105435137: status succeeded after adoption finished
I200419 05:47:07.365728 18264 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 5.567238901s, 3.90 MiB/s)
I200419 05:47:07.365925 17252 util/stop/stopper.go:539  quiescing
    --- FAIL: TestAllRegisteredImportFixture/tpcc (5.79s)
        all_test.go:95: sql: no rows in result set
            importing table warehouse
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1885962&tab=buildLog) on [master@666a0ac62832b6884bc6b039b4c944dbb42924aa](https://github.com/cockroachdb/cockroach/commits/666a0ac62832b6884bc6b039b4c944dbb42924aa):

```
I200420 05:29:43.567635 17236 kv/kvserver/stores.go:255  [n1] wrote 0 node addresses to persistent storage
I200420 05:29:43.847643 18364 jobs/registry.go:958  [n1] job 548176468599668737: status succeeded after adoption finished
I200420 05:29:44.022203 17964 ccl/workloadccl/fixture.go:446  imported 1017 B in district table (10 rows, 0 index entries, took 2.024698247s, 0.00 MiB/s)
I200420 05:29:45.028685 18093 jobs/registry.go:803  [n1] job 548176468486586369: stepping through state succeeded with error <nil>
I200420 05:29:45.222782 18093 jobs/registry.go:958  [n1] job 548176468486586369: status succeeded after adoption finished
I200420 05:29:45.259227 17968 ccl/workloadccl/fixture.go:446  imported 114 KiB in new_order table (9000 rows, 0 index entries, took 3.261270219s, 0.03 MiB/s)
I200420 05:29:45.469582 18252 jobs/registry.go:803  [n1] job 548176468522762241: stepping through state succeeded with error <nil>
I200420 05:29:45.594451 18252 jobs/registry.go:958  [n1] job 548176468522762241: status succeeded after adoption finished
I200420 05:29:45.614759 17967 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 3.616917924s, 0.42 MiB/s)
I200420 05:29:45.639429 18538 kv/kvserver/replica_command.go:397  [n1,s1,r38/1:/Table/54/{1-2}] initiating a split of this range at key /Table/54/1/0/1/1 [r81] (manual)
I200420 05:29:46.010494 18084 jobs/registry.go:803  [n1] job 548176468483899393: stepping through state succeeded with error <nil>
I200420 05:29:46.021566 18084 jobs/registry.go:958  [n1] job 548176468483899393: status succeeded after adoption finished
I200420 05:29:46.069645 18371 jobs/registry.go:803  [n1] job 548176468572405761: stepping through state succeeded with error <nil>
I200420 05:29:46.119544 18371 jobs/registry.go:958  [n1] job 548176468572405761: status succeeded after adoption finished
I200420 05:29:46.135203 17966 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 4.137472207s, 1.04 MiB/s)
I200420 05:29:46.249331 18334 jobs/registry.go:803  [n1] job 548176468575879169: stepping through state succeeded with error <nil>
I200420 05:29:46.309012 18334 jobs/registry.go:958  [n1] job 548176468575879169: status succeeded after adoption finished
I200420 05:29:46.326995 17969 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 4.328944975s, 1.80 MiB/s)
I200420 05:29:46.492807 18961 kv/kvserver/replica_command.go:397  [n1,s1,r61/1:/Table/55/{1-2}] initiating a split of this range at key /Table/55/1/0/1 [r82] (manual)
I200420 05:29:46.756429 18247 jobs/registry.go:803  [n1] job 548176468495368193: stepping through state succeeded with error <nil>
I200420 05:29:46.766973 18247 jobs/registry.go:958  [n1] job 548176468495368193: status succeeded after adoption finished
I200420 05:29:46.767229 18162 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 4.769082301s, 6.66 MiB/s)
I200420 05:29:46.872542 18395 kv/kvserver/replica_command.go:397  [n1,s1,r33/1:/Table/53/{1-2}] initiating a split of this range at key /Table/53/1/0/1/-3001/1 [r83] (manual)
I200420 05:29:47.430420 18190 jobs/registry.go:803  [n1] job 548176468417085441: stepping through state succeeded with error <nil>
I200420 05:29:47.440429 18190 jobs/registry.go:958  [n1] job 548176468417085441: status succeeded after adoption finished
I200420 05:29:47.440635 18163 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 5.443422305s, 3.99 MiB/s)
I200420 05:29:47.440821 17001 util/stop/stopper.go:539  quiescing
W200420 05:29:47.440893 20175 kv/kvserver/intentresolver/intent_resolver.go:745  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (5.67s)
        all_test.go:95: sql: no rows in result set
            importing table customer
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1888333&tab=buildLog) on [master@5ad50d887ad95e6b17b0d20fa98bc30319a67c0d](https://github.com/cockroachdb/cockroach/commits/5ad50d887ad95e6b17b0d20fa98bc30319a67c0d):

```
I200421 05:31:06.271204 18728 kv/kvserver/replica_command.go:397  [n1,s1,r64/1:/Table/6{0-1}] initiating a split of this range at key /Table/60/1 [r73] (manual)
I200421 05:31:06.271307 18674 jobs/registry.go:958  [n1] job 548459857896374273: status succeeded after adoption finished
I200421 05:31:06.296478 18728 kv/kvserver/replica_command.go:397  [n1,s1,r73/1:/Table/6{0/1-1}] initiating a split of this range at key /Table/60/2 [r74] (manual)
I200421 05:31:06.296677 18425 ccl/workloadccl/fixture.go:446  imported 1017 B in district table (10 rows, 0 index entries, took 730.035086ms, 0.00 MiB/s)
I200421 05:31:07.201900 17565 gossip/gossip.go:1538  [n1] node has connected to cluster via gossip
I200421 05:31:07.202043 17565 kv/kvserver/stores.go:255  [n1] wrote 0 node addresses to persistent storage
I200421 05:31:08.545856 18621 jobs/registry.go:803  [n1] job 548459857816059905: stepping through state succeeded with error <nil>
I200421 05:31:08.660010 18621 jobs/registry.go:958  [n1] job 548459857816059905: status succeeded after adoption finished
I200421 05:31:08.768499 18428 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 3.201645799s, 0.47 MiB/s)
I200421 05:31:09.857120 19509 kv/kvserver/replica_command.go:397  [n1,s1,r63/1:/Table/53/{1-2}] initiating a split of this range at key /Table/53/1/0/1/1 [r75] (manual)
I200421 05:31:10.083151 18594 jobs/registry.go:803  [n1] job 548459857870815233: stepping through state succeeded with error <nil>
I200421 05:31:10.135743 18594 jobs/registry.go:958  [n1] job 548459857870815233: status succeeded after adoption finished
I200421 05:31:10.148506 18427 ccl/workloadccl/fixture.go:446  imported 4.3 MiB in history table (30000 rows, 60000 index entries, took 4.581766017s, 0.94 MiB/s)
I200421 05:31:10.204474 18532 jobs/registry.go:803  [n1] job 548459857766940673: stepping through state succeeded with error <nil>
I200421 05:31:10.220548 18532 jobs/registry.go:958  [n1] job 548459857766940673: status succeeded after adoption finished
I200421 05:31:10.242911 18426 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 4.676562378s, 3.76 MiB/s)
I200421 05:31:10.310151 18717 jobs/registry.go:803  [n1] job 548459857992613889: stepping through state succeeded with error <nil>
I200421 05:31:10.344174 18717 jobs/registry.go:958  [n1] job 548459857992613889: status succeeded after adoption finished
I200421 05:31:10.536830 19448 kv/kvserver/replica_command.go:397  [n1,s1,r73/1:/Table/60/{1-2}] initiating a split of this range at key /Table/60/1/0/1 [r76] (manual)
I200421 05:31:11.050815 18728 jobs/registry.go:803  [n1] job 548459857950408705: stepping through state succeeded with error <nil>
I200421 05:31:11.060386 18728 jobs/registry.go:958  [n1] job 548459857950408705: status succeeded after adoption finished
I200421 05:31:11.060713 18431 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 5.45238081s, 5.83 MiB/s)
I200421 05:31:11.300710 18977 kv/kvserver/replica_command.go:397  [n1,s1,r38/1:/Table/58/{1-2}] initiating a split of this range at key /Table/58/1/0/1/-3001/1 [r77] (manual)
I200421 05:31:11.854698 18700 jobs/registry.go:803  [n1] job 548459857946902529: stepping through state succeeded with error <nil>
I200421 05:31:11.863308 18700 jobs/registry.go:958  [n1] job 548459857946902529: status succeeded after adoption finished
I200421 05:31:11.863589 18432 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 6.255469131s, 3.47 MiB/s)
I200421 05:31:11.863785 17522 util/stop/stopper.go:539  quiescing
W200421 05:31:11.863898 20579 kv/kvserver/intentresolver/intent_resolver.go:745  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (6.53s)
        all_test.go:95: sql: no rows in result set
            importing table item
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/workloadccl/allccl).TestAllRegisteredImportFixture failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1891450&tab=buildLog) on [master@3157234015595ba9b702222f90bfbb0b04994957](https://github.com/cockroachdb/cockroach/commits/3157234015595ba9b702222f90bfbb0b04994957):

```
I200422 05:24:48.583686 12614 gossip/gossip.go:1538  [n1] node has connected to cluster via gossip
I200422 05:24:48.583877 12614 kv/kvserver/stores.go:255  [n1] wrote 0 node addresses to persistent storage
I200422 05:24:48.589173 13554 jobs/registry.go:961  [n1] job 548741730820915201: status succeeded after adoption finished
I200422 05:24:48.590407 13400 ccl/workloadccl/fixture.go:446  imported 1.5 MiB in order table (30000 rows, 30000 index entries, took 2.211037044s, 0.68 MiB/s)
I200422 05:24:48.751891 13488 jobs/registry.go:961  [n1] job 548741730767863809: status succeeded after adoption finished
I200422 05:24:48.828940 13396 ccl/workloadccl/fixture.go:446  imported 53 B in warehouse table (1 rows, 0 index entries, took 2.450211012s, 0.00 MiB/s)
I200422 05:24:49.159634 13633 jobs/registry.go:806  [n1] job 548741730848964609: stepping through state succeeded with error <nil>
I200422 05:24:49.306783 13633 jobs/registry.go:961  [n1] job 548741730848964609: status succeeded after adoption finished
I200422 05:24:49.778711 13401 ccl/workloadccl/fixture.go:446  imported 114 KiB in new_order table (9000 rows, 0 index entries, took 3.399226976s, 0.03 MiB/s)
I200422 05:24:50.455342 14092 kv/kvserver/replica_command.go:397  [n1,s1,r44/1:/Table/54/{1-2}] initiating a split of this range at key /Table/54/1/0/1/1 [r70] (manual)
I200422 05:24:50.745813 13586 jobs/registry.go:806  [n1] job 548741730828288001: stepping through state succeeded with error <nil>
I200422 05:24:50.786157 13586 jobs/registry.go:961  [n1] job 548741730828288001: status succeeded after adoption finished
I200422 05:24:50.989639 13557 jobs/registry.go:806  [n1] job 548741730833924097: stepping through state succeeded with error <nil>
I200422 05:24:51.029404 13557 jobs/registry.go:961  [n1] job 548741730833924097: status succeeded after adoption finished
I200422 05:24:51.044954 13398 ccl/workloadccl/fixture.go:446  imported 18 MiB in customer table (30000 rows, 30000 index entries, took 4.665820841s, 3.76 MiB/s)
I200422 05:24:51.471606 14016 jobs/registry.go:806  [n1] job 548741732657889281: stepping through state succeeded with error <nil>
I200422 05:24:51.471947 14056 kv/kvserver/replica_command.go:397  [n1,s1,r43/1:/Table/57/{1-2}] initiating a split of this range at key /Table/57/1/0/1 [r71] (manual)
I200422 05:24:51.552614 14016 jobs/registry.go:961  [n1] job 548741732657889281: status succeeded after adoption finished
I200422 05:24:51.567154 13402 ccl/workloadccl/fixture.go:446  imported 7.8 MiB in item table (100000 rows, 0 index entries, took 5.187532823s, 1.50 MiB/s)
I200422 05:24:51.871373 13626 jobs/registry.go:806  [n1] job 548741730900377601: stepping through state succeeded with error <nil>
I200422 05:24:51.881913 13626 jobs/registry.go:961  [n1] job 548741730900377601: status succeeded after adoption finished
I200422 05:24:51.882226 13403 ccl/workloadccl/fixture.go:446  imported 32 MiB in stock table (100000 rows, 100000 index entries, took 5.484995105s, 5.79 MiB/s)
I200422 05:24:52.030564 13743 kv/kvserver/replica_command.go:397  [n1,s1,r33/1:/Table/55/{1-2}] initiating a split of this range at key /Table/55/1/0/1/-3001/1 [r72] (manual)
I200422 05:24:52.599661 13551 jobs/registry.go:806  [n1] job 548741730814361601: stepping through state succeeded with error <nil>
I200422 05:24:52.608751 13551 jobs/registry.go:961  [n1] job 548741730814361601: status succeeded after adoption finished
I200422 05:24:52.608996 13404 ccl/workloadccl/fixture.go:446  imported 22 MiB in order_line table (300343 rows, 300343 index entries, took 6.230195824s, 3.49 MiB/s)
I200422 05:24:52.609171 12507 util/stop/stopper.go:539  quiescing
W200422 05:24:52.609265 15491 kv/kvserver/intentresolver/intent_resolver.go:745  failed to gc transaction record: could not GC completed transaction anchored at /Local/Range/Table/55/1/RangeDescriptor: node unavailable; try another peer
    --- FAIL: TestAllRegisteredImportFixture/tpcc (6.45s)
        all_test.go:95: sql: no rows in result set
            importing table history
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:382
            github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
            github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
            	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357
            importing fixture
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl.ImportDataLoader.InitialDataLoad
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/fixture.go:322
            github.com/cockroachdb/cockroach/pkg/workload/workloadsql.Setup
            	/go/src/github.com/cockroachdb/cockroach/pkg/workload/workloadsql/workloadsql.go:39
            github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl.TestAllRegisteredImportFixture.func1
            	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/workloadccl/allccl/all_test.go:94
            testing.tRunner
            	/usr/local/go/src/testing/testing.go:909
            runtime.goexit
            	/usr/local/go/src/runtime/asm_amd64.s:1357

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestAllRegisteredImportFixture PKG=./pkg/ccl/workloadccl/allccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #47015 ccl/workloadccl/allccl: TestAllRegisteredImportFixture failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestAllRegisteredImportFixture.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/jobs/jobs.go
+++ b/pkg/jobs/jobs.go
@@ -846,6 +846,8 @@ func (sj *StartableJob) Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	jobCompletedOk := false
+
 	var r tree.Datums // stores a row if we've received one.
 	for {
 		// Alternate between receiving rows and sending them. Nil channels block.
@@ -868,6 +870,9 @@ func (sj *StartableJob) Run(ctx context.Context) error {
 			}
 		case toClient <- r:
 			r = nil
+			if jobCompletedOk {
+				return nil
+			}
 		case <-ctx.Done():
 			// Launch a goroutine to continue consuming results from the job.
 			if resultsFromJob != nil {
@@ -887,6 +892,11 @@ func (sj *StartableJob) Run(ctx context.Context) error {
 			return ctx.Err()
 		case err := <-errCh:
 			// The job has completed, return its final error.
+			if err == nil && r != nil {
+				// We still have data to send to the client.
+				jobCompletedOk = true
+				continue
+			}
 			return err
 		}
 	}

